executor/containerdexecutor/executor.go

```diff
@@ -100,6 +104,15 @@ func (w containerdExecutor) Exec(ctx context.Context, meta executor.Meta, root c
 	if system.SeccompSupported() {
 		opts = append(opts, seccomp.WithDefaultProfile())
 	}
+	if w.cgroupParent != "" {
+		var cgroupsPath string
+		if strings.Contains(w.cgroupParent, ":") {
+			cgroupsPath = w.cgroupParent + id
```

Comment:
Is this intentional that `cgroupParent` needs to end with separator. If yes, then we should validate it.